detox/src/client/actions/actions.js

```diff
@@ -14,6 +14,16 @@ class Action {
       throw new DetoxInternalError(`was expecting '${type}' , got ${JSON.stringify(response)}`);
     }
   }
+
+  /** @returns {boolean} */
+  get canBeConcurrent() {
```

Comment:
Rename suggestion (sponsored by naming-is-hard.com):
canBeConcurrent = not overly descriptive and drives logic negation in evaluation code.
How does `isAtomic` sound? Meaning, should be executed with no interruptions? This may be the Java dev in me talking...